src/Bravo3/ImageManager/Services/ImageManager.php

```diff
@@ -188,18 +188,20 @@ public function pull(Image $image)
     /**
      * Get meta information about the source image from the cache layer.
      *
-     * @param Image $image
+     * @param Image|string $image
      *
      * @return ImageMetadata
      */
-    public function getMetadata(Image $image)
+    public function getMetadata($image)
     {
         // If the $image is a variation, refer to its parent
         // for the metadata.
         if ($image instanceof ImageVariation) {
             $img_key = $image->getKey(true);
```

Comment:
I can't see where this breaks BC. This function still works as it always did, except you can now also pass a $key in place of an Image|ImageVariation.